content/guides/autodiscovery.md

```diff
@@ -48,155 +86,169 @@ By default, the Datadog Agent includes Autodiscovery support for:
 - Redis
 - Riak
 
-These are provided by the configuration templates in the Datadog Agent `conf.d/auto_conf` directory.
+Storing templates as local files is easy to understand and doesn't require an external service. The downside is that you must redeploy the Agent container each time you change, add, or remove templates. You may also have to maintain your own docker-dd-agent container if you want to add your own templates.
```

Comment:
Ok, got it. But I think no matter what, users must restart Agent containers to get new check configs, correct? i.e. there is no SIGHUP or similar way to tell Agent to reload the files?